### PR TITLE
Fixed #fixed incorrect use toString method.

### PR DESCRIPTION
**Pull request by mattisonchao**
Fixed #fixed incorrect use toString method.

Motivation
I found an incorrect use toString method. I think it's maybe a bug.

### CODE REVIEW

**Comment by mattisonchao** [comment]
Why is there no problem running the test locally?

**Review by sijie — CHANGES_REQUESTED** [review]
Can you provide more context on the problem you try to resolve?

**Comment by mattisonchao** [comment]
@sijie The return type of ``Base64.getDecoder().decode(msg)`` is ``byte[]``, calling toString will print the byte array address instead of the value.

**Review by aahmed-se — APPROVED** [review]

**Comment by mattisonchao** [comment]
@sijie  If you have time, can you help me review the code? : )

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java [hunk 1]
```diff
@@ -414,7 +414,7 @@ private int consumeFromWebSocket(String topic) {
                     LOG.debug("No message to consume after waiting for 5 seconds.");
                 } else {
                     try {
-                        System.out.println(Base64.getDecoder().decode(msg));
+                        System.out.println(Arrays.toString(Base64.getDecoder().decode(msg))); // print decode
```
Comment (eolivelli) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java:417]: Arrays.toString of a byte[] is still not very useful most of the times

in other commands in the pulsar-client we have an option to print byte[] in hex 
https://github.com/apache/pulsar/blob/27156e4866298c44e83724eeb10a4ee67317433c/pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java#L100

can you please implement the support for 'displayHex' ?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java (hunk 1 above)
Comment (mattisonchao) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java:417]: Ok, I will do that : )

Comment (mattisonchao) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java]: I have added support for `displayHex`

**Review by eolivelli — APPROVED** [review]
LGTM

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java [hunk 2]
```diff
@@ -414,8 +425,9 @@ private int consumeFromWebSocket(String topic) {
                     LOG.debug("No message to consume after waiting for 5 seconds.");
                 } else {
                     try {
-                        System.out.println(Base64.getDecoder().decode(msg));
-                    }catch(Exception e) {
+                        String output = interpretByteArray(displayHex, Base64.getDecoder().decode(msg));
```
Comment (eolivelli) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java:428]: this is the core of the change.
it is hidden inside the code reformat.

can you please revert the code reformat ?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java (hunk 2 above)
Comment (mattisonchao) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/client/cli/CmdConsume.java:417]: sure, I will fix it.

**Comment by mattisonchao** [comment]
@eolivelli PTAL

**Review by eolivelli — APPROVED** [review]
LGTM

**Comment by eolivelli** [comment]
@sijie now the patch is in better shape.
Please take a look when you have time

**Comment by mattisonchao** [comment]
@sijie  PTAL

**Review by sijie — APPROVED** [review]

**Pull request merged**